mkdocs.yml

```diff
@@ -22,7 +25,7 @@ nav:
     - Cloud: https://cloud.triggermesh.io
     - About: about.md
 theme:
-  name: mkdocs
+  name: "mkdocs"
```

Comment:
Those quotes aren't necessary.

Reply:
Haha I thought so, but their docs showed it so I got confused lol